Scripts/InstallPyPackages.py

```diff
@@ -80,16 +80,17 @@ def fixDictdifferNumpy():
 
     upgradePip()
 
-    installFromGit(owner='ikibalin', repo='cryspy', branch='transition-to-version-0.2')
+    #installFromGit(owner='pyinstaller', repo='pyinstaller', branch='develop')
```

Comment:
This commented code is not needed

Reply:
Just kept it to simplify switching from the master branch to another one for the cryspy library, when needed.